src/Symfony/Component/Form/Tests/Extension/Core/Type/UrlTypeTest.php

```diff
@@ -95,4 +103,47 @@ public function testSubmitNullUsesDefaultEmptyData($emptyData = 'empty', $expect
         $this->assertSame($expectedData, $form->getNormData());
         $this->assertSame($expectedData, $form->getData());
     }
+
+    /**
+     * @todo remove in 7.0
```

Comment:
As we won't remove the `default_protocol` feature in 7.0, there is no reason to remove this test if it covers a behavior that is not covered by other tests

Reply:
This test was inherited from `TextTypeTest` but it would have triggered a deprecation since it does not use `default_protocol`. Is there a better way to handle this?

Reply:
To me, the right fix would be that TextTypeTest should use the `getTestedType` and `getTestOptions` methods, as done in tests defined in BaseTypeTestCase, so that they respect the options defined by child test classes

Reply:
Indeed; done in https://github.com/symfony/symfony/pull/50922/commits/7956552402c76d490f21f1a0a15159f82b26ea9c